swaglyrics_backend/issue_maker.py

```diff
@@ -340,9 +336,9 @@ def update():
 
             if issue['status_code'] == 201:
                 print(f'Created issue on the GitHub repo for {song} by {artist}.')
-                return 'Lyrics for that song may not exist on Genius. ' \
+                return f'Lyrics for that song may not exist on Genius. ' \
                        'Created issue on the GitHub repo for {song} by {artist} to investigate ' \
-                       'further. \n{link}'.format(song=song, artist=artist, link=issue['link'])
+                       'further. \n{issue["link"]}'
```

Comment:
have you tested this?

Reply:
https://asciinema.org/a/pJvZ5fPVC3BNTIo2h6fObhnbL?t=38

Reply:
![image](https://user-images.githubusercontent.com/27063113/74734528-c00d4300-5274-11ea-8ab0-783bc6adab7d.png)
So yeah this doesn't work.

Reply:
Why it doesn't work is left as an exercise to the reader.

Reply:
I reverted the one in your example

Reply:
You need to add an f-string twice, let me change it

Reply:
![image](https://user-images.githubusercontent.com/26231126/74736333-65f78800-524a-11ea-8334-e015776156b0.png)

Reply:
yes, in fact, you don't need it in the first one. https://github.com/SwagLyrics/swaglyrics-backend/blob/8fb36e2d8511dd7593a977999a01db053ec2a4fd/swaglyrics_backend/issue_maker.py#L204 is an example